app/components/Sidebar.tsx

```diff
@@ -101,17 +102,17 @@ const Sidebar = () => {
                 <div className="space-y-5 relative">
                   <button
                     onClick={() => setFeedback(!feedback)}
-                    className="p-2 dark:bg-[#222] bg-[#EAEAEA] dark:text-slate-50 text-[#131313] text-x hover:opacity-90 rounded-lg w-full border-1 duration-300 transition-colors flex gap-5 items-center"
+                    className="p-2 dark:bg-[#222] bg-[#EAEAEA] dark:text-slate-50 text-[#131313] text-[0.8em] md:text-[1em] hover:opacity-90 rounded-lg w-full border-1 duration-300 transition-colors flex gap-5 items-center"
                   >
                     <VscFeedback />
                     Send Feedback
                   </button>
                   <div className="">{settingsModal && <SettingsModal />}</div>
                   <button
                     onClick={() => SetSettingsModal(!settingsModal)}
-                    className="flex p-3 dark:bg-[#222] bg-[#EAEAEA] rounded-lg justify-between items-center dark:text-slate-50 text-[#131313] hover:opacity-90 text-[14px] w-full"
+                    className="flex p-3 dark:bg-[#222] bg-[#EAEAEA] rounded-lg justify-between items-center text-[0.8em] md:text-[1em] dark:text-slate-50 text-[#131313] hover:opacity-90 text-[14px] w-full"
```

Comment:
I think you did this in the last commit you made for the pr

Reply:
Implementing responsive text sizes using breakpoints: 0.8em on mobile devices and 1em from tablet to desktop for improved readability and visual consistency.